doc/transform.ipynb

```diff
@@ -2335,8 +2335,14 @@
    "cell_type": "markdown",
```

Comment:
<p><code>CFSR_thetao = xr.open_dataset(URL,engine='netcdf4')</code>  do you need to specify <code>engine</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>I think I was working with a previous version that required me to specify engine. Here's how I download the data now:</p><p> </p><pre class="ql-syntax" spellcheck="false">cfsr_thetao_nc = netCDF4.Dataset(URL)
cfsr_thetao = xr.open_dataset(xr.backends.NetCDF4DataStore(cfsr_thetao_nc))
</pre>